calcom/cal.com
# fix: improve cancellation email of reschedule RR booking

## What does this PR do?



- Fixes #17253 (GitHub issue number)
- Fixes CAL-4585 (Linear issue number - should be visible at the bottom of the GitHub issue description)



[Screencast from 21-11-24 12:49:55 AM IST.webm](https://github.com/user-attachments/assets/dba3e896-ea1d-400a-8327-0a0125a5ae1d)


## How should this be tested?

1. Create a Round Robin Event and make sure that Reschedule with same Round-Robin host is toggled off in the Event Advanced Settings Tab.
2. Book the event.
3. Now reschedule the booking to a different date.
4. Check the mails and notice that the cancellation mail sent to the old round robin host shows old date and it also shows the details of the member the rescheduled booking has been reassigned to.

## Mandatory Tasks (DO NOT REMOVE)
